packages/frontend/ci/configFromEnvironment.js

```diff
@@ -0,0 +1,70 @@
+const assert = require("assert");
+
+const Environments = require("../../../features/environments.json");
+const { parseBooleanFromShell } = require("../src/config/envParsers");
+
+const Config = {
+  BRANCH: process.env.BRANCH,
+  CLOUDFLARE_BASE_URL: process.env.CLOUDFLARE_BASE_URL,
+  CONTEXT: process.env.CONTEXT,
+  DEBUG_BUILD: process.env.DEBUG_BUILD,
+  DEPLOY_PRIME_URL: process.env.DEPLOY_PRIME_URL,
+  IS_DEVELOPMENT: process.env.NODE_ENV === "development",
+  IS_NETLIFY: parseBooleanFromShell(process.env.NETLIFY),
+  IS_RENDER: parseBooleanFromShell(process.env.RENDER),
+  IS_PULL_REQUEST: parseBooleanFromShell(process.env.IS_PULL_REQUEST),
+  NEAR_WALLET_ENV: process.env.NEAR_WALLET_ENV,
+  RENDER: process.env.RENDER,
+  RENDER_EXTERNAL_URL: process.env.RENDER_EXTERNAL_URL,
+  RENDER_GIT_COMMIT: process.env.RENDER_GIT_COMMIT,
+  REVIEW_ID: process.env.REVIEW_ID,
+  SENTRY_AUTH_TOKEN: process.env.SENTRY_AUTH_TOKEN,
+  SENTRY_DSN: process.env.SENTRY_DSN,
+  SENTRY_RELEASE:
+      process.env.SENTRY_RELEASE ||
+      (process.env.RENDER &&
```

Comment:
Should be using `parseBooleanFromShell` here rather than directly referencing `process.env.RENDER`